app/views.py

```diff
@@ -58,4 +58,19 @@ def register():
 @app.route('/trips')
 @login_required
 def trips():
-    return render_template('trips.html')
\ No newline at end of file
+    return render_template('trips.html')
+
+
+
+@app.route('/trips/create', methods=['GET', 'POST'])
+@login_required
+def create_trip():
+    form = TripForm()
+    if form.validate_on_submit():
+        trip = trip(name=form.name.data, destination=form.destination.data)
+        user.set_password(form.password.data)

```

Comment:
We don't want to set the user's password, each time they want to create a trip. Just delete this line